src/modules/Tasks.js

```diff
@@ -0,0 +1,14 @@
+const taskses = [{ task: 'Double-tap to edit' }, { task: "Drag'n drop to reorder your list" }, { task: 'Manage all your lists in one place' }, { task: 'Resync to clear out the old' }];
```

Comment:
- [ ] Please you are required to create a list of objects with description/task, completed, and index
<img width="981" alt="image" src="https://user-images.githubusercontent.com/37901469/171513182-87b573d3-eb56-4866-bc7e-e71177a11705.png">

Reply:
This issue is well resolved! ✔️